velox/exec/Spill.cpp

```diff
@@ -16,9 +16,18 @@
 
 #include "velox/exec/Spill.h"
 #include "velox/common/file/FileSystems.h"
+#include "velox/serializers/PrestoSerializer.h"
 
 namespace facebook::velox::exec {
 
+// Spilling currently uses the default PrestoSerializer which by default
+// serializes timestamp with millisecond precision to maintain compatibility
+// with presto. Since velox's native timestamp implementation supports
+// nanosecond precision, we use this serde option to ensure the serializer
+// preserves precision.
+static const serializer::presto::PrestoVectorSerde::PrestoOptions
+    kDefaultSerdeOptions = {true};
```

Comment:
`kDefaultSerdeOptions{.useLosslessTimestamp = true}` would be more clear and does not need the constructor.

Reply:
unable to use this with an explicit constructor but added a comment for greater visibility

Reply:
You can remove the constructor IMO, for options a plain struct looks more suitable.

Reply:
@Yuhta The compiler seems to complain about not being able to find a constructor, probably because the base class is a pure virtual class